pkg/operator/workload/helper.go

```diff
@@ -0,0 +1,43 @@
+package workload
+
+import (
+	"fmt"
+	"github.com/aquasecurity/trivy-operator/pkg/kube"
+	"github.com/go-logr/logr"
+	"golang.org/x/net/context"
+	appsv1 "k8s.io/api/apps/v1"
+	batchv1 "k8s.io/api/batch/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+func Inspect(ctx context.Context, resource client.Object, or kube.ObjectResolver, scanOnlyCurrentRevisions bool, log logr.Logger) (bool, error) {
```

Comment:
Can we improve the name? Inspect doesn't give any idea about what this func is doing?

Reply:
@josedonizetti sure , the other thing on my mind was `validate` happy for suggestion

Reply:
@chen-keinan I think what we are doing here is `skipping`, which I also prefer because would change the check to `if skipping` instead of `if !skipping`. So I'm thinking:

```
skip, err = workload.SkipProcessing() // or ShouldSkip()
if skip {
return
}
```

wdyt?